orc8r/cloud/deploy/terraform/orc8r-aws/variables.tf

```diff
@@ -134,6 +141,26 @@ variable "thanos_worker_groups" {
 
 }
 
+variable "blue_green_worker_groups" {
+  # Check the docs at https://github.com/terraform-aws-modules/terraform-aws-eks
+  # for the complete set of valid properties for these objects. This worker group exists
+  # in order to increase the capacity of the EKS resources to accomodate a second instance
+  # of the Orc8r/NMS
+  description = "Worker group configuration for EKS. Default value is 1 worker group consisting of 5 t3.medium instances."
+  type        = any
+  default = [
+    {
+      name                 = "wg-1"
+      instance_type        = "t3.medium"
+      asg_desired_capacity = 5
+      asg_min_size         = 1
+      asg_max_size         = 5
+      autoscaling_enabled  = false
```

Comment:
Should we consider enabling autoscaling_enabled to 'true' with desired capacity reduced to 3 or 4 (as the number of supported pods per node)? This will enable us to have a desired capacity that is lesser than max, but have the ability to automatically scale when the 'Green' cluster is deployed.

Reply:
We will start with the default size as we discussed while enabling autoscaling in a follow up PR.